build.sbt

```diff
@@ -21,7 +21,7 @@ libraryDependencies ++= Seq(
   javaWs,
   javaJpa,
   "org.eclipse.persistence" % "eclipselink" % "2.6.2",
-  "com.github.ONSdigital" % "dp-dd-backend-model" % "develop-SNAPSHOT",
+  "com.github.ONSdigital" % "dp-dd-backend-model" % "feature~metadata-SNAPSHOT",
```

Comment:
Yes, there will be a little dance to make sure nothing breaks: merge this and the API changes first, then merge the model, and finally update both projects to revert back to using develop-SNAPSHOT.